content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -17,11 +18,11 @@ A database transaction refers to a sequence of read/write operations that are _g
 
 Prisma provides the following options for using transactions:
 
-- **Nested writes**: use the Prisma Client API to process multiple operations on one or more related records inside the same transaction.
-- **Batch / bulk transactions**: to process one or more operations in bulk via `updateMany`, `deleteMany`, and `createMany`.
-- **The `$transaction` calls**:
-  - `$transaction<R>(queries: PrismaPromise<R>[]): Promise<R[]>`: Pass an array of Prisma Client queries to be executed sequentially inside of a transaction.
-  - `$transaction<R>(fn: (prisma: PrismaClient) => R): R`: Pass a function that can contain user code including Prisma Client queries, non-Prisma code and other control flow to be executed in a transaction.
+- [Nested writes](#nested-writes): use the Prisma Client API to process multiple operations on one or more related records inside the same transaction.
+- [Batch / bulk transactions](#batchbulk-operations): process one or more operations in bulk via `updateMany`, `deleteMany`, and `createMany`.
+- The Prisma Client `$transaction` API:
+  - [Sequential operations](#sequential-prisma-client-operations): pass an array of Prisma Client queries to be executed sequentially inside a transaction, using `$transaction<R>(queries: PrismaPromise<R>[]): Promise<R[]>`.
+  - [Interactive transactions](#interactive-transactions-in-preview) (in Preview): pass a function that can contain user code including Prisma Client queries, non-Prisma code and other control flow to be executed in a transaction, using `$transaction<R>(fn: (prisma: PrismaClient) => R): R`: P
```

Comment:
Or, if we want shorter version, since specific options are described below:

```suggestion
  - [Interactive transactions](#interactive-transactions-in-preview) (in Preview): pass a function that can contain user code including Prisma Client queries, non-Prisma code and other control flow to be executed in a transaction, using `$transaction<R>(fn: (prisma: PrismaClient) => R, options?: object): R`: P
```